packages/features/vault/src/components/custom-tool-tip.tsx

```diff
@@ -0,0 +1,55 @@
+import { Box, styled } from '@mui/material';
+import {
+  formatNumberAsPercent,
+  getDateString,
+} from '@notional-finance/helpers';
+import { H5 } from '@notional-finance/mui';
+import { ONE_WEEK } from '@notional-finance/shared-config';
+import { FormattedMessage } from 'react-intl';
+import { TooltipProps } from 'recharts';
+import { messages } from '../messages';
+
+const ToolTipBox = styled(Box)(
+  ({ theme }) => `
+  background: ${theme.palette.common.white};
+  padding: ${theme.spacing(2)};
+  text-align: left;
+  box-shadow: ${theme.shape.shadowStandard}
+`
+);
+
+export const CustomTooltip = (props: TooltipProps<number, string>) => {
```

Comment:
Is it possible to merge the tool tip components? We can have a prop of `tooltipContent: FormattedMessage[]` and each one gets wrapped in an `H5` component?

I know one of them gets bolded or whatever, we can maybe always just bold the last value or none at all?